### DISCUSSION TITLE
Hide either horizontal or vertical lines in grid

### DISCUSSION THREAD

**Discussion by write2varun** [Blazor-Diagrams/Blazor.Diagrams#485] [Q&A] [answered]
Hi is there a way to hide horizontal or vertical line in grid. I tried adding css but it gave an error. i saw the code the style is embedded in the code. Is it possible to convert that into a css style. 
Thank you
Regards,
varun

**Comment by zHaytam** [comment, marked as answer]
Currently, no sorry. Can you please create an issue from the discussion?

In the meantime, you can just copy the code of the Grid widget and modify it.

**Comment by write2varun** [comment]
Thank you
